src/Services/QueuedJobService.php

```diff
@@ -680,7 +681,7 @@ public function runJob($jobId)
                 }
             } catch (Exception $e) {
                 // okay, we'll just catch this exception for now
-                $this->getLogger()->error($e->getMessage());
+                $this->getLogger()->debug($e->getMessage());
```

Comment:
I had done so after reading https://docs.silverstripe.org/en/4/developer_guides/debugging/error_handling/

However I misread it as "debug and warn" should be through logger and "user_error" for errors.